src/openms/source/ANALYSIS/MAPMATCHING/FeatureGroupingAlgorithmKD.cpp

```diff
@@ -210,10 +210,10 @@ namespace OpenMS
             if (input_maps[k][m].getMZ() >= partition_start &&
                 input_maps[k][m].getMZ() < partition_end)
             {
-              tmp_input_maps[k].push_back(input_maps[k][m]);
+              tmp_input_maps[k].push_back(&(input_maps[k][m]));
             }
           }
-          tmp_input_maps[k].updateRanges();
+          //tmp_input_maps[k].updateRanges();
```

Comment:
remove unused code if unnecessary.

Reply:
removing this might cause compile error, we are going to check this again